### DISCUSSION TITLE
cypress returns request header or cookie too large

### DISCUSSION THREAD

**Discussion by huseyin-tumer** [cypress-io/cypress#27284] [Questions and Help]
Hello guys!
when using cypress, my website login page redirectes to another url after login, but the another url returns 400 bad request, request header or cookie too large.

When i do it manually with my own crome, it not returns. it runs nice.

- Node version: 20.4.0
- Npm version: 9.7.2
- Cypress version: 12.17.1
- OS: Windows 11

What should i do?
![Screenshot 2023-07-12 at 20 01 40](https://github.com/cypress-io/cypress/assets/45070155/bf04daae-cace-447b-b7e6-cb9f010f3a3d)

Thanks

**Comment by JeanRessouche** [comment]
Same here :(

**Comment by rahulmr** [comment]
Same here with below code, any workarounds ?
```
PS D:\projects\nodejs\douglas> npx cypress --version
Cypress package version: 13.13.0
Cypress binary version: 13.13.0
Electron version: 27.3.10
Bundled Node version:
18.17.1
PS D:\projects\nodejs\douglas> node -v
v20.14.0
PS D:\projects\nodejs\douglas> npm -v
10.8.2
PS D:\projects\nodejs\douglas>
```
source code

```javascript
describe('template spec', () => {
  it('passes', () => {
    cy.visit('https://www.douglas.de/de')
    cy.visit('https://douglas.de/de')
    
  })
})
```

**Reply by MikeMcC399** [reply]
@rahulmr 

Yours is probably a different problem.

If you are employed by the owner of the website domain `douglas.de` then you should contact your website support colleagues to explain why your request is rejected.

If you do not have any relationship with `douglas.de` or permission to test their website, then please read https://docs.cypress.io/guides/references/best-practices#Visiting-External-Sites
